css/pico.amber.css

```diff
@@ -1966,17 +1955,17 @@ article {
 }
 article > header,
 article > footer {
-  margin-right: calc(var(--pico-block-spacing-horizontal) * -1);
-  margin-left: calc(var(--pico-block-spacing-horizontal) * -1);
+  margin-inline-start: calc(var(--pico-block-spacing-horizontal) * -1);
+  margin-inline-end: calc(var(--pico-block-spacing-horizontal) * -1);
   padding: calc(var(--pico-block-spacing-vertical) * 0.66) var(--pico-block-spacing-horizontal);
   background-color: var(--pico-card-sectioning-background-color);
 }
 article > header {
   margin-top: calc(var(--pico-block-spacing-vertical) * -1);
   margin-bottom: var(--pico-block-spacing-vertical);
   border-bottom: var(--pico-border-width) solid var(--pico-card-border-color);
-  border-top-right-radius: var(--pico-border-radius);
-  border-top-left-radius: var(--pico-border-radius);
+  border-start-start-radius: var(--pico-border-radius);
```

Comment:
This might be confusing but `border-*-*-radius` has also `inline`/`block` corresponding. 

ref. https://developer.mozilla.org/en-US/docs/Web/CSS/border-end-end-radius